mlflow/tensorflow.py

```diff
@@ -252,6 +267,7 @@ def save_model(
     :param pip_requirements: {{ pip_requirements }}
     :param extra_pip_requirements: {{ extra_pip_requirements }}
     """
+    _raise_deprecation_warning()
```

Comment:
I didn't add `_raise_deprecation_warning` in `log_model` because it internally calls `save_model`.